cmd/govim/testdata/scenario_tempmodfile/modfile_changes.txt

```diff
@@ -0,0 +1,37 @@
+# A simple test that verifies we get a diagnostic for changes
+# that need to be made to our go.mod file
+
+skip 'Pending a fix for golang.org/issues/36247'
```

Comment:
Since I didn't review this PR until now 36247 is now closed, should we update deps to latest and drop the skip? :)

Reply:
Yes I'm going to rebase this on top of #713 and drop the skip.